### PR TITLE
NOTIK: Implemented topic info endpoint

### PR DESCRIPTION
**Pull request by ghost**
Implemented endpoint `/api/topics/{topic}/info` to get partition info for a topic

### CODE REVIEW

File: http/snoop/src/main/kotlin/kafkasnoop/routes/topics.kt [hunk 1]
```diff
@@ -64,3 +66,50 @@ fun NormalOpenAPIRoute.topics(kafkaClientFactory: KafkaClientFactory) {
...
+                )
+            )
+        )
+    ) { params ->
+        kafkaClientFactory
+            .createConsumer().use {
+                respond(
+                    it.partitionsFor(params.topic).run {
+                        val topicPartitions = this.map { p -> TopicPartition(params.topic, p.partition()) }
+                        val beggingOffsets = it.beginningOffsets(topicPartitions)
```
Comment (anton-subbotin) [inline http/snoop/src/main/kotlin/kafkasnoop/routes/topics.kt]: typo? (begging -> beginning)

**Review by anton-subbotin — APPROVED** [review]
1. One minor comment 
2. The repository doesn't include any tests, is it something doable or worth thinking of?

File: http/snoop/src/main/kotlin/kafkasnoop/routes/topics.kt (hunk 1 above)
Comment (ghost) [reply http/snoop/src/main/kotlin/kafkasnoop/routes/topics.kt]: this is the second time I'm doing this 😄

**Comment by ghost** [comment]
Last time I tried to mock some classes to write unit tests, but it stated taking too long and I quit.
These endpoints didn't have any tests. I think this calls for some integration tests and at some point I'll need to spend the time to write them

**Pull request merged**